cobalt/h5vcc/h5vcc_crash_log.cc

```diff
@@ -174,5 +174,29 @@ void H5vccCrashLog::SetPersistentSettingWatchdogCrash(bool can_trigger_crash) {
   if (watchdog) watchdog->SetPersistentSettingWatchdogCrash(can_trigger_crash);
 }
 
+void H5vccCrashLog::LogEvent(std::string event) {
+  watchdog::Watchdog* watchdog = watchdog::Watchdog::GetInstance();
+  if (watchdog) watchdog->LogEvent(event);
```

Comment:
Pretty sure style guide requires brackets around this if conditional. Here and throughout.

Reply:
`pre-commit` passes this way, anyway I've changed it with return value change.